### ISSUE TITLE
Overlay Srollbars stops Materialize Parallax Effect

### ISSUE THREAD

**Issue by dalbitresb12** [KingSora/OverlayScrollbars#68]
Hi:

I'm creating a site using Materialize, and I recently decided to add custom scrollbars. I tried OverlayScrollbars but I noticed that when OverlayScrollbars is working, the Parallax effect stops working. You can see it here: https://jsfiddle.net/dalbitresb/hu87csgk/39/
At the end of the HTML there's a script with the `OverlayScrollbars(document.querySelectorAll("body"), {});` commented, un-comment it and you will notice the Parallax Effect stops working.

**Comment by KingSora** [comment]
Good day!

I've investigated into your problem, and I've found a solution.

But first let me explain the problem:
The plugin wraps it's content inside of a generated DOM structure. So the element which is responsible for the scrolling changes. If you initialize `OverlayScrollbars` to the `body-element`, the `window-element` will never fire a `scroll` event, because the plugin changes scroll-responsible element. 

The solution is to tell the `window-element` about our scrolling:
Just initialize the plugin like this:
```javascript
OverlayScrollbars(document.querySelectorAll("body"), {
  callbacks : {
    onScroll : function() { 
      var event = document.createEvent('HTMLEvents');
      event.initEvent("scroll", true, false);
      window.dispatchEvent(event);
    }
  }
});
```
There _could_ be bugs, because `materialize` is checking the `scrollTop` value which should be your current scroll-offset, but if you initialize `OverlayScrollbars` this value is always 0. - I've tested it and haven't found any noticable bugs though.
